lib/msal-common/src/config/ClientConfiguration.ts

```diff
@@ -46,7 +46,7 @@ export type ClientConfiguration = {
     telemetry?: TelemetryOptions,
     serverTelemetryManager?: ServerTelemetryManager | null,
     persistencePlugin?: ICachePlugin | null,
-    serializableCache?: ISerializableTokenCache | null
+    serializableCache?: ISerializableTokenCache | null,        
```

Comment:
nit: can you delete all the extra space you added in here and elsewhere in this review?